runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -154,6 +149,7 @@
 
 
     private SslContext sslContext;
+    private final Map<CorfuMsgType, String> timerNameCache = new HashMap<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert this Map to an EnumMap. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1640)